doc/classes/SplitContainer.xml

```diff
@@ -77,6 +80,18 @@
 		</signal>
 	</signals>
 	<constants>
+		<constant name="AUTO_SNAP_NONE" value="0" enum="AutoSnap">
+			The dragger will never snap automatically to any of the container's edges.
+		</constant>
+		<constant name="AUTO_SNAP_FIRST" value="1" enum="AutoSnap">
+			The dragger will snap automatically to the edge on which the first child lies when [member split_offset] is set to less than a third of its minimum size. When [member vertical] is [code]true[/code] this is the top edge, otherwise it is the left edge.
+		</constant>
+		<constant name="AUTO_SNAP_SECOND" value="2" enum="AutoSnap">
+			The dragger will snap automatically to the edge on which the second child lies when [member split_offset] is set to greater than the container's size minus a third of the child's minimum size. When [member vertical] is [code]true[/code] this is the bottom edge, otherwise it is the right edge.
```

Comment:
```suggestion
			The dragger will snap automatically to the edge on which the second child lies when [member split_offset] is set to greater than the container's size minus a third of the child's minimum size. If [member vertical] is [code]true[/code], this is the bottom edge. Otherwise, it is the right edge.
```